Tasks/Gradle/testlib/tasklib-wrapper.ts

```diff
@@ -0,0 +1,158 @@
+// Redirects calls to vsts-task-lib to testlib if in a test environment.
```

Comment:
Modules do not implement interfaces, so the wrapper re-calls the same method on the production task library or the test library, as appropriate.